kafka-quotas-plugin.iml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't think we should be checking this file in. IIUC it's an intellij thing, which intellij can recreate when a project is opened.